src/main/export/export.ts

```diff
@@ -1,53 +1,71 @@
 /* eslint-disable no-plusplus */
 import { BrowserWindow } from 'electron';
-import { writeFileSync } from 'fs';
+import { existsSync, writeFileSync } from 'fs';
 import path, { join } from 'path';
-import { secondToTimestamp, padZeros } from '../timeUtils';
+import ffprobe from 'ffprobe';
```

Comment:
Check if this breaks packaging, I'm pretty sure we were using the static version for this reason

Reply:
Changed to use `fluent-ffmpeg` instead of `ffprobe` directly, tested packaging and it is A-OKAY! (on my end (on Windows) )